packages/backend-app-api/config.d.ts

```diff
@@ -32,6 +34,16 @@ export interface Config {
        * unless you configure credentials for service calls.
        */
       dangerouslyDisableDefaultAuthPolicy?: boolean;
+      rateLimit?: {
```

Comment:
> Hm I wonder where this should best live. I'd say backend.rateLimit.unauthorized perhaps (no auth. Then we can add more backend.rateLimit.catalog.blah or whatnot down the line. I was thinking that maybe this isn't really strictly always auth related per se? 🤔
It makes sense to me if we want to be able to have different rate limit settings in the future. Also, putting it outside the `auth` configuration makes sense 🙂

> Also, should these be marked @visibility frontend just so that it's easier to make well-behaved clients? Or should they just work away and then respect response headers instead?
I would say that it seems to be redundant as we have the same config info in the headers, but I would love to hear your opinion, what do you think?

Reply:
Suggestions implemented 🙌🏻